### PR TITLE
coala_as_Git_Hook.html: Change sentence

### PR DESCRIPTION
**Pull request by divyam3897**
Changes the sentence,
Using git hooks coala
can be executed
automatically, ensuring
your code follows your
quality requirements
to Using a git hook,
coala can be executed
automatically, ensuring
your code follows your
quality requirements.

Closes https://github.com/coala/documentation/issues/117

### CODE REVIEW

File: Users/coala_as_Git_Hook.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ﻿Git Hooks
 =========
```
Comment (Adrianzatreanu) [inline Users/coala_as_Git_Hook.rst]: first of all, i believe its .rst, not .html (in commits hortlog).

also, what is that  body wrapping? text should be wrapped @72 chars, not at 20! it looks really bad.

**Comment by sils1297** [comment]
ack https://github.com/coala/documentation/pull/139/commits/bc649272ca5efd4e5501ca3896950515a41fae2d

**Comment by sils1297** [comment]
@rultor merge

**Comment by sils1297** [comment]
Hey, this is your first contribution, right? Congrats and welcome to the coalaians! :)

You can help us a lot by filling out <http://coala.io/newform>!

Rultor will now automatically release your changes as a prerelease so users can experience the goodness of your changes right now.

See also https://github.com/coala-analyzer/coala/wiki/coala-Steam-Offer btw.

**Comment by sils1297** [comment]
It looks like your PR is out of date and needs a rebase. [This page](http://coala.io/git) may help you to get started on this. (*Do not press the update branch button. It's evil.*)

Add the original coala repository as a remote:

```
git remote add upstream https://github.com/coala/documentation.git
```

Then go to your branch and rebase it onto our master:

```
git checkout <yourbranch>
git fetch upstream
git rebase --interactive upstream/master
```

Now push it up. You will need to do a force push because you change history here (do not do this on master branches :)):

```
git push --force
```

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix it up for you - just tell us :).

Ping us on [our gitter channel](https://coala.io/chat) if you need help!

**Comment by sils1297** [comment]
ack https://github.com/coala/documentation/pull/139/commits/cf67bccd726fdeb6036b98ad6458aeee79826551

**Comment by sils1297** [comment]
@rultor merge

**Pull request merged**